kibana.d.ts

```diff
@@ -40,6 +40,19 @@ export namespace Legacy {
   export type SavedObjectsService = LegacyKibanaServer.SavedObjectsService;
   export type Server = LegacyKibanaServer.Server;
 
+  export type InitPluginFunction = (server: Server) => void;
+  export interface UiExports {
+    injectDefaultVars: (server: Server) => { [key: string]: any };
+  }
+
+  export interface PluginFactoryConfig {
```

Comment:
Is it possible to put this interface in a `.d.ts` file where the implementation of the `kibana.Plugin` class is and the re-exporting that type here? I prefer it being close to the implementation so we can easily find where the type information is coming from.

Reply:
good idea